### DISCUSSION TITLE
Add compartment in Lung

### DISCUSSION THREAD

**Discussion by Supercn01** [Open-Systems-Pharmacology/Forum#1699] [Models]
Hi everyone
    I want add a new compartment in Lung. Could anyone provide me some tutorial materials? I will every appreciated!

Thanks!

**Comment by nicospinu** [comment]
At [esqLABS](https://esqlabs.com/), we have developed a course on PBPK for inhalation, which will soon be released. If you are interested in being granted early access to it, could you please write to us at training@esqlabs.com to book a quick call? 
Alternatively, the work by the research group of Dr. Andrea Edginton at the University of Waterloo is open access and available at https://github.com/Open-Systems-Pharmacology/Inhalation-model. 
If you check our catalog of courses [here](https://esqlabs.shop/collections/online-courses), you can find courses focused on the technical side of OSP Suite as well. Depending on your affiliation, we can grant free access to it.
I hope that helps!
